### DISCUSSION TITLE
View all players with /who

### DISCUSSION THREAD

**Discussion by veselin-ugrinov** [TrinityCore/TrinityCore#26286] [Q&A] [answered]
Hi guys. Sorry for my noobish question but stil... I want all users in my server to be able to see all online players with /who command. So far this is only possible for the GM's, other users see only players with the same faction ... Is this possible ?

**Comment by Jildor** [comment, marked as answer]
Check rbac permission in auth DB

**Reply by veselin-ugrinov** [reply]
I dont know which ID coresponds to that i want ...

**Reply by Jildor** [reply]
```sql
SELECT * FROM `rbac_permissions` WHERE `id` IN (28,194,195);

    id  name                       
------  ---------------------------
    28  See two side who list      
   194  Role: Sec Level Moderator  
   195  Role: Sec Level Player   
```

```sql
SELECT * FROM `rbac_linked_permissions` WHERE `linkedId`=28;

    id  linkedId  
------  ----------
   194          28
```

if you want set that rbac_permission for all you should change to rbac role player instead of moderator:
```sql
UPDATE `rbac_linked_permissions` SET `id`=195 WHERE `linkedId`=28;
```

**Comment by Aokromes** [comment]
orrr
https://community.trinitycore.org/topic/13962-trouble-with-your-trinity-install-starting-login-readme-1st-faqs/?tab=comments#comment-60049

**Reply by ghost** [reply]
(just to reorder the lines from that page, in numerical order, as well as applying TC SQL codestyle)

```sql
INSERT INTO `rbac_linked_permissions` (`id`, `linkedId`) VALUES
(195, 24), -- Two side faction characters on the same account
(195, 25), -- Say chat between factions
(195, 26), -- Channel chat between factions
(195, 27), -- Two side mail interaction
(195, 28), -- See two side who list
(195, 29), -- Add friends of other faction
(195, 36); -- Allow to use whispers off
```

**Reply by veselin-ugrinov** [reply]
worked like a charm ... thank you, kind sir ! ;)

**Comment by ex1dreamer** [comment]
Hi, any know issues with assigning the same permissions (i.e. 24,25,26,27,28,29,36) for both roles 195 and 194? 
I'm having issue "no player named xxx is currently playing" after INSERT INTO command from the post above

**Reply by ghost** [reply]
I see. Well, that happens to me too, even on a GM/Admin account, between a Horde and Alliance character.
Note: the message appears when I try to add the offline alliance character to the horde guild, without any cross-faction settings enabled.
- Maybe the character has to be online to be added to the guild, or
- maybe the character needs to belong to a different account?

**Reply by ex1dreamer** [reply]
Nope, it seems completely broken. Doesn't work for on-line users either. Both belong to Horde. 
Command .whisper ON doesn't help

**Reply by ghost** [reply]
Odd. Not sure what to do to reproduce it, then.

Could you write a list of things to do on a test account, so we may be able to replicate or reproduce your issue?

**Reply by ex1dreamer** [reply]
maybe later. droped and recreated rbac_linke_permission from the backup. problem disappeared.

**Reply by ghost** [reply]
OK, fair enough. Some times, that is just as good, the same way as "Have you tried turning it off and on again?" ^^
 =^.^=

**Comment by Greyisttot** [comment]
mark
